polynote-frontend/polynote/data/codec.ts

```diff
@@ -170,7 +170,7 @@ export class DataWriter {
 
     writeInt64(value: number) {
         this.ensureBufSize(this.buffer.byteLength + 8);
-        this.dataView.setBigInt64(this.offset, value);
+        this.dataView.setBigInt64(this.offset, BigInt(value));
```

Comment:
`value` is already `number` which means it already can't represent a `Long`... what's the point?

Reply:
In this case the point was that `setBigInt64` was broken without this 😛 but you're right that it's incomplete. 

Should we change the int64 codec from `Codec<number>` to `Codec<bigint>`, then?

Reply:
It would make sense to me to change it to `Codec<bigint>`. Also, apparently vega now supports `bigint` (from what I could tell) so it might make sense to upgrade vega as well.

Reply:
done